.pre-commit-config.yaml

```diff
@@ -24,8 +24,12 @@ repos:
       - id: mypy
         exclude: ^docs/conf.py
 
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.2.1
+  - repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.5
     hooks:
-      - id: prettier
-        args: [--prose-wrap=always, --print-width=88]
+      - id: mdformat
+        args: [--number, --wrap=88]
```

Comment:
I'd argue that removing `--number` arguement would be more in the spirit of Black (minizing diffs is great, [docs](https://mdformat.readthedocs.io/en/stable/users/style.html#ordered-lists) explain this).

Reply:
I literally copied and pasted the configuration you suggested in that other issue but OK :laughing: . Looking into this, I agree omitting the `--number` argument is better.

Reply:
Haha yeah I know. The initial suggestion was just meant to be as close to the Prettier style that was in use as possible. I thought you may not want to change the style in the scope of this PR. But seeing that the diffs are pretty huge anyways, maybe you do :smile: ?

Reply:
In fact, you may want to consider omitting `--wrap=88` for the same reason and adopt [semantic line breaks](https://sembr.org/). That change however requires a lot of manual effort unless you migrate to it gradually.